topics/assembly/tutorials/debruijn-graph-assembly/data-library.yaml

```diff
@@ -1,8 +1,24 @@
 ---
-libraries:
+destination:
+  name: GTN - Material
+  type: library
+  description: Galaxy Training Network Material
+  synopsis: Galaxy Training Network Material. See https://training.galaxyproject.org
+items:
+- name: Assembly
+  description: DNA sequence data has become an indispensable tool for Molecular Biology
+    & Evolutionary Biology. Study in these fields now require a genome sequence to
+    work from. We call this a 'Reference Sequence.' We need to build a reference for
+    each species. We do this by Genome Assembly. De novo Genome Assembly is the process
+    of reconstructing the original DNA sequence from the fragment reads alone.
+  items:
   - name: assembly_data
-    files:
-      - url: https://zenodo.org/record/582600/files/mutant_R1.fastq
-        file_type: "fastqsanger"
-      - url: https://zenodo.org/record/582600/files/mutant_R2.fastq
-        file_type: "fastqsanger"
+    items:
+    - url: https://zenodo.org/record/582600/files/mutant_R1.fastq
+      src: url
+      ext: fastqsanger
+      info: ''
```

Comment:
@bedroesb do you have the time to manually add useful `info` to these files? I think that would be great.

Reply:
@bgruening I can add info available on zenodo, but this can take some time.